pandas/tests/frame/test_aggregate.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
```

Comment:
these go in pandas/tests/apply/test_frame_apply.py

there are already extensive tests, so find a good place to exercise the empty case

we have these, so model after

```
(pandas) jreback@lhs2:~/pandas-dev$ grep -r empty pandas/tests/apply/|grep def
pandas/tests/apply/test_frame_transform.py:def test_transform_empty_listlike(float_frame, ops, frame_or_series):
pandas/tests/apply/test_frame_transform.py:def test_transform_empty_dictlike(float_frame, ops, frame_or_series):
pandas/tests/apply/test_frame_transform.py:def test_transform_empty_dataframe():
pandas/tests/apply/test_series_apply.py:def test_apply_empty_integer_series_with_datetime_index():
pandas/tests/apply/test_series_apply.py:def test_map_empty(index):
pandas/tests/apply/test_frame_apply.py:def test_apply_empty(float_frame):
pandas/tests/apply/test_frame_apply.py:def test_apply_with_reduce_empty():
pandas/tests/apply/test_frame_apply.py:def test_apply_funcs_over_empty(func):
pandas/tests/apply/test_frame_apply.py:def test_nunique_empty():
pandas/tests/apply/test_frame_apply.py:def test_apply_empty_infer_type():
pandas/tests/apply/test_frame_apply.py:def test_apply_empty_list_reduce():
```